scripts/install/requestrr.sh

```diff
@@ -0,0 +1,182 @@
+#!/bin/bash
+# Requestrr installation
+# Author: Brett
+# Copyright (C) 2021 Swizzin
+# Licensed under GNU General Public License v3.0 GPL-3 (in short)
+#
+#   You may copy, distribute and modify the software as long as you track
+#   changes/dates in source files. Any modifications to our software
+#   including (via compiler) GPL-licensed code must also be made available
+#   under the GPL along with build & install instructions.
+#shellcheck source=sources/functions/requestrr
+#shellcheck source=sources/functions/appconfigs
+#shellcheck source=sources/functions/users
+. /etc/swizzin/sources/functions/requestrr
+. /etc/swizzin/sources/functions/appconfigs
+. /etc/swizzin/sources/functions/users
+user=$(_get_master_username)
+_requestrr_download
+
+echo_progress_start "Extracting archive"
+unzip -q /tmp/requestrr.zip -d /opt/ >> "$log" 2>&1
+rm /tmp/requestrr.zip
+mv /opt/requestrr* /opt/requestrr
+echo_progress_done "Archive extracted"
+
+echo_progress_start "Creating requestrr user and setting permssions"
+useradd --system -d /opt/requestrr/ requestrr
+chown -R requestrr:requestrr /opt/requestrr/
+chmod u+x /opt/requestrr/Requestrr.WebApi
+echo_progress_done "Requestrr user has been created & permissions set."
+
+echo_progress_start "Applying Requestrr config..."
+
+echo_progress_start "Grabbing list of apps and integrating with Requestrr"
+_get_sonarr_vars
+_get_radarr_vars
+echo_progress_done "Apps have been added and will be added to Requestrr config."
+
+cat > /opt/requestrr/SettingsTemplate.json << CFG
+{
+  "Authentication": {
+    "Username": "",
+    "Password": "",
+    "PrivateKey": "[PRIVATEKEY]"
+  },
+  "ChatClients": {
+    "Discord": {
+      "BotToken": "",
+      "ClientId": "",
+      "StatusMessage": "!help",
+      "TvShowRoles": [],
+      "MovieRoles": [],
+      "MonitoredChannels": [],
+      "EnableRequestsThroughDirectMessages": false,
+      "AutomaticallyNotifyRequesters": true,
+      "NotificationMode": "PrivateMessages",
+      "NotificationChannels": [],
+      "AutomaticallyPurgeCommandMessages": false,
+      "DisplayHelpCommandInDMs": true
+    }
+  },
+  "DownloadClients": {
+    "Ombi": {
+      "Hostname": "",
+      "Port": 3579,
+      "ApiKey": "",
+      "ApiUsername": "",
+      "BaseUrl": "",
+      "UseSSL": false,
+      "Version": "3"
+    },
+    "Overseerr": {
+      "Hostname": "",
+      "Port": 5055,
+      "ApiKey": "",
+      "DefaultApiUserID": "",
+      "UseSSL": false,
+      "Version": "1"
+    },
+    "Radarr": {
+      "Hostname": "${r_address}",
+      "Port": ${r_port},
+      "ApiKey": "${r_key}",
+      "BaseUrl": "/${r_base}",
```

Comment:
This should just be standard
```suggestion
      "BaseUrl": "/radarr",
```